spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/data/mongo/MongoReactiveDataAutoConfiguration.java

```diff
@@ -85,4 +89,20 @@ public MappingMongoConverter mappingMongoConverter(MongoMappingContext context,
 		return mappingConverter;
 	}
 
+	@Bean
+	@ConditionalOnMissingBean
+	public ReactiveGridFsTemplate reactiveGridFsTemplate(
+			ReactiveMongoDatabaseFactory reactiveMongoDbFactory,
+			MappingMongoConverter mappingMongoConverter,
+			DataBufferFactory dataBufferFactory) {
+		return new ReactiveGridFsTemplate(dataBufferFactory, reactiveMongoDbFactory,
+				mappingMongoConverter, null);
+	}
+
+	@Bean
+	@ConditionalOnMissingBean
+	public DataBufferFactory dataBufferFactory() {
```

Comment:
Can we flip those two methods? Required dependencies are defined before so I'd like to keep it that way.